requirements.txt

```diff
@@ -1,4 +1,11 @@
 --index-url https://pypi.python.org/simple/
 
 -e .
-https://github.com/matplotlib/basemap/archive/master.zip    # pip install of basemap currently fails: https://github.com/matplotlib/basemap/issues/251
+
+# pip install of older basemap fails: see https://github.com/matplotlib/basemap/issues/251
+# Pin to basemap 1.2.2, the last release before they changed their package
+# structure.  Newer versions of basemap are once again installable via pip
+# (see https://pypi.org/search/?q=basemap), however they require a newer
+# version of numpy and that is a can of worms that we don't want to open
+# at this time.
+https://github.com/matplotlib/basemap/archive/refs/tags/v1.2.2rel.zip
```

Comment:
Should this be moved into `setup.py` like is done [here](https://github.com/cisagov/cyhy-mailer/blob/9a62890a2e9a98b656717ece94fd89209aa32ad3/setup.py#L57)? This is not a good place for package dependencies.

Reply:
Yes, that is a good call.  Please have a look at https://github.com/cisagov/cyhy-reports/pull/71/commits/8e5aba35f8d8785660f355152f53b045773e4dcb.